server/services/__integration__/s3dataLoader.spec.js

```diff
@@ -0,0 +1,10 @@
+const s3dataLoader = require('../s3dataLoader')
+const extraInfoService = require('../extraInfoService')
+
+describe('/S3DataLoader test', () => {
+  test('loads the manifest file', async () => {
+    const data = await s3dataLoader()
+    const matchingData = extraInfoService.featuresAtPoint(data, 374676.7543833861, 164573.87856146507, true)
```

Comment:
Can these numbers be named? I appreciate it's just test data but can be helpful for if someone if trying to understand the code.